iam-policy-autopilot-access-denied/src/commands/apply.rs

```diff
@@ -82,6 +82,7 @@ impl super::service::IamPolicyAutopilotService {
             sort_statements(&mut merged_statements);
 
             let policy_doc = crate::types::PolicyDocument {
+                id: Some("IamPolicyAutopilot".to_string()),
```

Comment:
For an existing policy, are we currently dropping the `Id` if there was one set? Now we would replace it with `IamPolicyAutopilot`? Do we need to check if the existing policy has an `Id` set?